README.md

```diff
@@ -53,8 +53,9 @@ The notebooks `c2q_transfer_learning_*.ipynb` require the Python libraries Penny
 
 The notebooks `q2c_transfer_learning.ipynb` and  `q2q_transfer_learning.ipynb` require the library Strawberry Fields with the TensorFlow backend.
 
-For the requirements of the code in `quantum_processors\` see the dedicated README file `quantum_processors\README.md`.
+All notebooks require the package versions specified in the `requirements.txt` to run on a CUDA device.
```

Comment:
```suggestion
Notebooks can be run using the package versions specified in the `requirements.txt` file.
```
Perhaps this way could present a nice "fall-back" for anyone experiencing compatibility issues, regardless of using CUDA or not.